### DISCUSSION TITLE
Does it have rollback similar to ones found in renpy

### DISCUSSION THREAD

**Discussion by ThanksTnx** [nathanhoad/godot_dialogue_manager#546] [Q&A]
Thinking of experimenting with Godot and this dialogue manager. Just curious if it can do this?
When you scroll with the middle mouse it moves the dialogue along.

**Comment by nathanhoad** [comment]
That dialogue manager itself is stateless but you can implement something like this in your game by keeping track of each `next_id` in the chain.
